instances/technique/lightSheetFluorescenceMicroscopy.jsonld

```diff
@@ -4,11 +4,13 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/lightSheetFluorescenceMicroscopy",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
+  "definition": "Lightsheet fluorescence microscopy is an fluorescence microscopy technique that uses a thin sheet of light to excite only fluorophores within the plane of illumination.",
```

Comment:
"a fluorescence" not "an fl." - "Lightsheet fluorescence microscopy is a fluorescence microscopy..."